src/lib/chips/chip-list.ts

```diff
@@ -317,4 +506,138 @@ export class MdChipList implements AfterContentInit, OnDestroy {
 
     return false;
   }
+
+  _setSelectionByValue(value: any, isUserInput: boolean = true) {
+    this._clearSelection();
+    this.chips.forEach((chip) => chip.deselect());
+    const isArray = Array.isArray(value);
```

Comment:
I wouldn't add the extra `isArray` variable here and would just use `isArray()` in the `if` block